config/sample_settings/general.json

```diff
@@ -1,13 +1,20 @@
 {
-    "monitored_accounts": [
+    "tooling_environment": {
+        "Name": "Tooling Account",
```

Comment:
Let's also name it like "Tooling Account [dev]" to demonstrate the config approach we suggest to future users

Reply:
Added